indico/modules/events/timetable/controllers/manage.py

```diff
@@ -217,6 +224,27 @@ def _process_DELETE(self):
         delete_timetable_entry(self.break_.timetable_entry)
         return '', 204
 
+    @no_autoflush
+    def _move_entry(self, session_block_id):
+
+        entry = self.break_.timetable_entry
+
+        if session_block_id is None:
+            update_timetable_entry(entry, {
+                'parent': None,
+                'start_dt': entry.start_dt
+            })
+        else:
+            target_block = self.event.get_session_block(int(session_block_id))
```

Comment:
The `session_block_id` is defined as an int in the schema, so you'll always get it as an int here:

```suggestion
            target_block = self.event.get_session_block(session_block_id)
```